pkg/daemon/config_drift_monitor.go

```diff
@@ -17,13 +17,19 @@ import (
 )
 
 // Outermost error type for config drift errors
-type configDriftErr error
+type configDriftErr struct {
```

Comment:
This comment is intended for other reviewers to explain why this particular change was done: We discovered that the way we were using `errors.As()` in this case was incorrect. Wrapping the error in a struct like this is the preferred way to represent a simple custom error type as opposed to doing something like `type customError error`.